### DISCUSSION TITLE
Move pretixPOS license

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3325] [Q&A] [answered]
Hello Pretix team.
Can I move a pretixPOS license to another Android device?
Is there a limitation on how many times this can be done ?

**Comment by raphaelm** [comment]
Yes, the license is always for "x devices used simultaneously with the same server", so you can switch devices as often as you like.

**Reply by alemao8** [reply]
Thank you for the immediate response.
Is there any trial license I can use before purchasing one, or a demo environment to have a look of the functionality?

**Reply by raphaelm** [reply, marked as answer]
Yes, you can use a free test account of pretix Hosted (https://pretix.eu) to play around with pretixPOS
